data/planet.lua

```diff
@@ -92,7 +113,7 @@ local planets = {
         },
     },
     {
-        type = "space-location",
+        type = "planet", -- You should be able to land on asteroids in the asteroid belt.
```

Comment:
*Small asteroids, if you've ever played SE and went to orbit - This is what I'm thinking (I've played SE and been to orbit)

Reply:
land on ceres

Reply:
Ah, ok - and also the other asteroids in the belt (such as ceres) can be landed on.
[https://spaceexploration.miraheze.org/wiki/File:AsteroidField.PNG](This) is what I was thinking, but not from map view.